ocfweb/docs/docs/staff/tips/templates.md

```diff
@@ -0,0 +1,8 @@
+[[!meta title="Email Templates"]]
+
+General email response templates can be found at [templates.ocf.berkeley.edu](https://templates.ocf.berkeley.edu) for purposes such as sending requests for DNS to the University hostmaster and notifying banned users. Templates should be used when applicable because they are designed to provide all necessary information while remaining cordial to the recipient.
```

Comment:
Can you wrap these lines to 80 characters?

Reply:
It's rather rare for us to ban users, the templates user-how-to-get-vhost (for people who come asking "how do I get mygroup.berkeley.edu") and user-alumni-reset-pass (for alumni who can't reset their passwords online) are more commonly used examples of templates